src/Selenium2Library/keywords/_element.py

```diff
@@ -479,17 +479,27 @@ def simulate(self, locator, event):
     def press_key(self, locator, key):
         """Simulates user pressing key on element identified by `locator`.
 
-        `key` is either a single character, or a numerical ASCII code of the key
-        lead by '\\\\'.
+        `key` is either a single character, a numerical ASCII code of the key,\
+         or a NAMED KEY as described at https://selenium.googlecode.com/git/docs/api/py/webdriver/selenium.webdriver.common.keys.html
 
         Examples:
-        | Press Key | text_field   | q |
-        | Press Key | login_button | \\\\13 | # ASCII code for enter key |
-        """
-        if key.startswith('\\') and len(key) > 1:
-            key = self._map_ascii_key_code_to_key(int(key[1:]))
-        #if len(key) > 1:
-        #    raise ValueError("Key value '%s' is invalid.", key)
+        | Press Key | text_field   | q        |                                               |
+        | Press Key | login_button | \\\\13   | # ASCII code for Enter key (DEPRECATED)       |
+        | Press Key | login_button | 10       | # ASCII code for Return key                   |
+        | Press Key | nav_console  | ARROW_UP | # selenium.webdriver.common.keys ARROW_UP KEY |
+        
+        NAMED KEY value is new in Selenium2Library 1.7.3.
+        """
+        if len(key) > 1:
+            if key.startswith('\\'):
+                self._warn("Press Key: Escaped ASCII codes are deprecated. Use plain numeric value: '%s'" % (key[1:]))
+                key = self._map_ascii_key_code_to_key(int(key[1:]))
+            else:
+                try:
+                    key = (key.isdecimal() and self._map_ascii_key_code_to_key(int(key))) or\
+                          ((not key.isdecimal()) and self._map_named_key_code_to_special_key(key))
+                except:
```

Comment:
Plain` except` is not a good idea. It can mask bugs and prevents `SystemExit` and `KeyboardInterrupt` to stop execution.